stacks/single-server/appliance-unlock.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Very much might be wrong here, but using the word 'appliance' for the single-server setup stuff seems like it might be confusing to end-users. I realize it probably fits the criteria for being called an appliance, but perhaps another name might be more appropriate? ('instance-unlock' or just 'set-admin-password' or something to that effect?)

Reply:
Could, but it's targeted at AWS Marketplace compliance, so it's only ever going to be used when it's serving a role *as* an appliance, which is why I named it like that. Don't feel strongly about it, though.

Reply:
In other words, the only time it'll ever be used is in the post-deployment Marketplace notes 
when a user is explicitly told to use it.

Reply:
Perhaps having it in the main docker repo is the wrong place for it then, if it's only ever expected to be used in the docker appliance. It might then make more sense to inject it in as needed.

Reply:
Maybe, but I'm going to replace it with a GENERALIZED password recovery script as soon as I can manage it, so it'll go in the same place.